main.go

```diff
@@ -16,6 +16,70 @@
 
 package main
 
+import (
+	"cmd-exclude-prefixes-k8s/internal/prefixcollector"
+	"cmd-exclude-prefixes-k8s/internal/utils"
+	"context"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/jaeger"
+	"github.com/networkservicemesh/sdk/pkg/tools/signalctx"
+	"github.com/networkservicemesh/sdk/pkg/tools/spanhelper"
+)
+
+// Config - configuration for cmd-exclude-prefixes-k8s
+type Config struct {
+	ExcludedPrefixes   []string             `desc:"List of excluded prefixes" split_words:"true"`
+	ConfigMapNamespace string               `default:"default" desc:"Namespace of kubernetes config map" split_words:"true"`
+	KubeConfig         utils.KubeConfigPath `desc:"Path to kubeconfig file"`
+}
+
 func main() {
+	// Capture signals to cleanup before exiting
+	ctx := signalctx.WithSignals(context.Background())
+
+	closer := jaeger.InitJaeger("prefix-service")
+	defer func() { _ = closer.Close() }()
+
+	span := spanhelper.FromContext(context.Background(), "Start prefix service")
+	defer span.Finish()
+
+	// Get clientSetConfig from environment
+	config := &Config{}
+	if err := envconfig.Usage("", config); err != nil {
```

Comment:
I think we should add a name here and on process(line 56)

Reply:
Added.